checklists/afd_checklist.en.json

```diff
@@ -186,7 +186,8 @@
             "guid": "17ba124b-127d-42b6-9322-388d5b2bbcfc",
             "id": "A01.29",
             "severity": "High",
-            "link": "https://learn.microsoft.com/azure/web-application-firewall/ag/application-gateway-waf-request-size-limits#request-body-inspection"
+            "graph" : "resources | where type =~ 'microsoft.network/frontdoorwebapplicationfirewallpolicies' | where properties['policySettings']['requestBodyCheck'] =~ "Enabled"| extend compliant = properties['policySettings']['requestBodyCheck'] =~ 'Enabled' | distinct id, name, compliant",
```

Comment:
Please use single quotes (instead of double quotes) inside of the ARG query string

Reply:
Updated with single quotes, thanks Jose! :)